platform/nodejs/index.js

```diff
@@ -201,6 +201,51 @@ function reset() {
     snfe.reset();
 }
 
+/******************************************************************************/
+
+let pslInitialized = false;
+let staticNetFilteringEngineInstance = null;
+
+class StaticNetFilteringEngine {
+    constructor() {
+        if ( staticNetFilteringEngineInstance !== null ) {
+            throw new Error('Only a single instance is supported.');
+        }
+
+        staticNetFilteringEngineInstance = this;
+
+        this._context = new FilteringContext();
+    }
+
+    async useLists(lists) {
+        await useRawLists(lists);
```

Comment:
I have some ideas about how to decide whether to call `useRawLists()` or `useCompiledLists()` but I'll put that in a separate patch.